src/scss/themes/_live.scss

```diff
@@ -21,9 +21,17 @@
             }
         }
 
-        .o-teaser__heading a:visited {
-            color: oColorsGetPaletteColor('white-80');
+        .o-teaser__heading a {
+            &:focus,
+            &:hover {
+                color: oColorsMix(white, crimson, 90);
+            }
+
+            &:visited {
+                color: oColorsGetPaletteColor('white-80');
```

Comment:
Is "visited" still supposed to be `white-80`? Because `white-80` still has some FT pink in it

Reply:
I don't know. I'm aware that it was a fix that Arjun added in https://github.com/Financial-Times/o-teaser/pull/98.

Reply:
"I probably completely made that up." – Arjun, November 2018

Reply:
We've updated visited to match hover and focus.

Reply:
We also noticed the imperceptible difference for the default style while looking into this.

https://github.com/Financial-Times/o-teaser/blob/5d2b6fa253612b94de53ce3f1780395b647182b3/src/scss/elements/_default.scss#L46-L53